src/hb-ot-color-colr-table.hh

```diff
@@ -39,13 +39,21 @@ namespace OT {
 
 struct LayerRecord
 {
+  operator hb_ot_color_layer_t () const
+  {
+    hb_ot_color_layer_t result = {0};
+    result.glyph = glyphId;
+    result.color_index = colorIdx;
```

Comment:
Since `hb_ot_color_layer_t` is public and never changes, just initialize in one line?